sh/gtf/gxa_references.conf

```diff
@@ -77,11 +83,11 @@ solanum_tuberosum ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/sola
 sorghum_bicolor ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/sorghum_bicolor/Sorghum_bicolor.Sorghum_bicolor_NCBIv3.RELNO.gtf.gz
 sus_scrofa ftp://ftp.ensembl.org/pub/release-RELNO/gtf/sus_scrofa//Sus_scrofa.Sscrofa11.1.RELNO.gtf.gz
 tetraodon_nigroviridis ftp://ftp.ensembl.org/pub/release-RELNO/gtf/tetraodon_nigroviridis/Tetraodon_nigroviridis.TETRAODON8.RELNO.gtf.gz
-theobroma_cacao ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/theobroma_cacao/Theobroma_cacao.Criollo_cocoa_genome_V2.RELNO.gtf.gz
```

Comment:
@pinin4fjords could you check if ISL is using Theobroma cacao criollo or Theobroma cacao with no variety set? Thanks. This changed happened a while back when E! Plants moved to criollo as default, but they have again reinstated the none criollo one.

Reply:
Current 'active' configuration in ISL is:

```
reference=Theobroma_cacao.Criollo_cocoa_genome_V2.dna.toplevel.fa.gz
cdna_file=Theobroma_cacao.Criollo_cocoa_genome_V2.cdna.all.44.fa.gz
gtf_file=Theobroma_cacao.Criollo_cocoa_genome_V2.44.gtf.gz
```

... but I'll check the config file to see what's being downloaded.